planner/core/casetest/integration_test.go

```diff
@@ -1578,6 +1578,44 @@ func TestIndexJoinRangeFallback(t *testing.T) {
 	}
 }
 
+func TestFixControl45132(t *testing.T) {
+	store := testkit.CreateMockStore(t)
+	tk := testkit.NewTestKit(t, store)
+	tk.MustExec(`use test`)
+	tk.MustExec(`create table t (a int, b int, key(a))`)
+	for i := 0; i < 10; i++ {
+		tk.MustExec(`insert into t values (1, 1), (1, 1), (1, 1), (1, 1), (1, 1), (1, 1), (1, 1), (1, 1), (1, 1), (1, 1)`)
```

Comment:
It can merge one SQL. you can reference the code from executor/test/indexmergereadtest/index_merge_reader_test.go:1201

Reply:
updated